netutils/password.py

```diff
@@ -206,25 +211,25 @@ def encrypt_type7(unencrypted_password, salt=None):
         >>>
     """
     if not salt:
-        salt = random.randrange(0, 15)  # nosec
-    encrypted_password = "%02x" % salt  # pylint: disable=consider-using-f-string
+        salt = random.randrange(0, 15)  # type: ignore # nosec
+    encrypted_password = "%02x" % salt  # type: ignore # pylint: disable=consider-using-f-string
     for i, _ in enumerate(unencrypted_password):
-        hex_password = "%02x" % (ord(unencrypted_password[i]) ^ XLAT[salt])  # pylint: disable=consider-using-f-string
+        hex_password = "%02x" % (ord(unencrypted_password[i]) ^ XLAT[salt])  # type: ignore # pylint: disable=consider-using-f-string
         encrypted_password += hex_password
-        salt += 1
-        if salt == 51:
-            salt = 0
+        salt += 1  # type: ignore
+        if salt == 51:  # type: ignore
+            salt = 0  # type: ignore
```

Comment:
If we use a different variable name for the changed form of `salt`, we can get by without all the ignore comments